### ISSUE TITLE
Broken Auto Attack

### ISSUE THREAD

**Issue by Shragenator** [SallyWhitemane/Mograine#1108]
Randomly when playing, Auto Attack will cease to work.  The player character will be able to still use melee abilities but auto attacks will not function.   When your character ends up in this state the only way I have found to fix it is logging off entirely. 

This has happened to myself quite a few times in the last week (on death knight), and many others have said they have experienced this as well. 

I have not found any source of the issue, so I cannot reproduce it at the moment.

**Comment by danniellos94** [comment]
Yes, I think even autoattack macro doesn't work hard to say when the problem occurs. Relog seems to solve the problem tho.

**Comment by scrublama** [comment]
Never happened me
What class, spec, race change are you?

**Comment by Shragenator** [comment]
DK in either blood or frost specs.   Human race changed to Nelf.  

come to think of it, this has only been happening since I most recently race changed

**Comment by Shragenator** [comment]
UPDATE:  I'm pretty sure it's noggenfogger elixir causing this!  Doesn't happen every time but I can reproduce it by spamming noggenfogger.   Maybe this also happens when using some other consumables?
